### PR TITLE
Fix function declaration in if statement

### PR DESCRIPTION
**Pull request by spicyj**
Firefox doesn't like these and throws.

cc @zpao (for lint), @jimfb (for sync)

### CODE REVIEW

**Comment by jimfb** [comment]
👍

**Comment by facebook-github-bot-2** [comment]
@spicyj updated the pull request.

File: scripts/bench/extract-component.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ function print(outerComponent) {
               debugger;
```
Comment (yordis) [inline scripts/bench/extract-component.js]: Is this right?

Comment (spicyj) [inline scripts/bench/extract-component.js]: Yes, this is only used interactively and that's an error case.

File: scripts/bench/extract-component.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ function print(outerComponent) {
         var values = keys.map((childKey) => renderedChildren[childKey]);
 
         if (keys.length) {
-          function dump(children) {
+          var dump = function(children) {
```
Comment (chicoxyzzy) [inline scripts/bench/extract-component.js]: it's better to write `var dump = function dump(children) {` to keep function names in stack trace

Comment (spicyj) [inline scripts/bench/extract-component.js]: The code have here is fine for what I use it for.

**Pull request merged**